deps/rabbit/src/rabbit_connection_tracking.erl

```diff
@@ -450,18 +264,11 @@ lookup(Name, [Node | Nodes]) ->
     end.
 
 lookup_internal(Name, Node) ->
-    case rabbit_feature_flags:is_enabled(tracking_records_in_ets) of
-        true -> lookup_ets(Name, Node);
-        false -> lookup_mnesia(Name, Node)
-    end.
+    lookup_ets(Name, Node).
```

Comment:
There are multiple functions in this file which just call `*_ets`.
I think they can be all inlined meaning the `*_ets` functions can be deleted.
WDYT?

Reply:
Done